### PR TITLE
Don't process multiple times

### PR DESCRIPTION
**Pull request by rhlsthrm**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/adapters/database/db/migrations/20230213141356_root-message-sent-status.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+-- migrate:up
+ALTER TABLE root_messages
+ADD COLUMN sent_time timestamp with time zone,
```
Comment (preethamr) [inline packages/adapters/database/db/migrations/20230213141356_root-message-sent-status.sql:3]: Minor:
We have been storing timestamps as integers. This deviates from it.

**Review by preethamr — APPROVED** [review]
Have one minor comment.

**Review by liu-zhipeng — APPROVED** [review]

File: packages/adapters/database/db/migrations/20230213141356_root-message-sent-status.sql (hunk 1 above)
Comment (rhlsthrm) [reply packages/adapters/database/db/migrations/20230213141356_root-message-sent-status.sql]: Yep fixed this.

**Comment by rhlsthrm** [comment]
@preethamr pls re-review there have been a few changes.

File: packages/agents/lighthouse/src/tasks/processFromRoot/operations/processFromRoot.ts [hunk 2]
```diff
@@ -124,6 +127,39 @@ export const processSingleRootMessage = async (
     });
   }
 
+  if (rootMessage.sentTaskId) {
```
Comment (preethamr) [inline packages/agents/lighthouse/src/tasks/processFromRoot/operations/processFromRoot.ts:130]: This info is not being store in DB.
`rootMessage.sentTaskId`, `rootMessage.sentTimestamp`, and `rootMessage.relayerType` are not upserted into DB here

**Review by preethamr — CHANGES_REQUESTED** [review]
Needs further changes.

**Comment by preethamr** [comment]
@rhlsthrm I put in a fix with:
https://github.com/connext/monorepo/commit/6dc0a118591223b8ad5c84f922f5222296a5195a

Pls merge if good.

**Comment by rhlsthrm** [comment]
> @rhlsthrm I put in a fix with: [6dc0a11](https://github.com/connext/monorepo/commit/6dc0a118591223b8ad5c84f922f5222296a5195a)
> 
> Pls merge if good.

Ah wow! Thank you for this.

**Pull request merged**